crates/primitives/src/log.rs

```diff
@@ -20,6 +21,12 @@ pub struct Log {
     pub data: Bytes,
 }
 
+impl From<AlloyLog> for Log {
```

Comment:
Do we need this? Should we re-use types, or not possible? @mattsse @DaniPopes

Reply:
Good guess @DaniPopes. Tried to switch it, but it didn't work, left it as it was.